libs/support/test-support/src/lib/map/components/action/draw-action/vl-map-draw-polygon-action.tester.js

```diff
@@ -17,12 +17,14 @@ export class VlMapDrawPolygonActionTester extends VlMapDrawActionTester {
             for (let i = 0; i < coordinates.length; i++) {
                 const coordinate = coordinates[i];
                 const pixel = await map.getPixelFromCoordinate([coordinate.x, coordinate.y]);
+                await actions.move({origin: map, x: pixel.x, y: pixel.y}).pause(1000); // wacht bij een move/klik aangezien te snel klikken problematisch is (foutief
```

Comment:
vermoed dat hier hier wsl `/ foutief` of `(foutief)` moest staan?

Reply:
Copy paste foutje, idd :)